src/RestApi/StoreApi/Schemas/CartShippingRateSchema.php

```diff
@@ -214,6 +220,15 @@ public function get_item_response( $package ) {
 				]
 			),
 			'items'          => $items,
+			'name'           => apply_filters(
+				'woocommerce_shipping_package_name',
+				( $package['index'] > 0 ) ?
+					/* translators: %d: shipping package number */
+					sprintf( _x( 'Shipping %d', 'shipping packages', 'woo-gutenberg-products-block' ), ( $package['index'] + 1 ) ) :
```

Comment:
In 3ddf8b70a5750c1d2c7a02a054089c90ae6bfdb5 I renamed this from `index` to `key`, because its values might not always be ordered integers.